### ISSUE TITLE
Cookie banner uses default user agent colour and margins for content

### ISSUE THREAD

**Issue by 36degrees** [alphagov/govuk-design-system#1886]
## Description of the issue

The cookie banner (in both the Design System's own cookie consent implementation and the examples) uses the default user agent colour and margins for paragraphs within the content.

This is visible if you change the default text colour in your browser (jn this example, the default text colour has been changed to orange):

![Screenshot 2021-09-20 at 16 53 44](https://user-images.githubusercontent.com/121939/134033627-3071853e-5d30-43c9-bd59-810dc335ccc4.png)

The examples in GOV.UK Frontend consistently apply the `govuk-body` class to all paragraphs, so this issue is not currently visible there.

![Screenshot 2021-09-20 at 16 57 09](https://user-images.githubusercontent.com/121939/134034187-8f77f00e-fc79-4057-8bf4-8d2dd9a82f41.png)

## Expected design

- The text colour of the cookie banner content should be 'GOV.UK black' (`#0b0c0c`)
- The margins on the paragraphs in the cookie banner content should be consistent with our other paragraph styles

## Actual design

- The text colour of the cookie banner content uses the user agent default text colour (normally `#000000`) and margins

**Comment by 36degrees** [comment]
I think technically, [according to F24, this fails WCAG 1.4.3, 1.4.6 and 1.4.8](https://www.w3.org/TR/WCAG20-TECHS/F24.html).

We should either update `govuk-cookie-banner__content` to style child paragraphs, or update the examples (and guidance?) to include `govuk-body` on all paragraphs?

The latter seems easy to miss though – as evidenced by the fact we've only just spotted this ourselves…!

**Comment by vanitabarrett** [comment]
I've raised PRs for the approach of adding `govuk-body` to the cookie banner examples and implementation in the design system. I've also raised a PR to expand our examples in the govuk-frontend review app so we have visual examples that use `headingHtml` and `html` to test against.

If we're happy with this approach, I'll create a follow-up ticket to remove the default styling to make it more obvious if a user  forgets the `govuk-body` class, and include it in v4.0.0.

**PRs:**

[Add cookie banner examples using headingHtml and html](https://github.com/alphagov/govuk-frontend/pull/2353)
[Add missing govuk-body classes to cookie banner](https://github.com/alphagov/govuk-design-system/pull/1891)
